pkg/kubelet/dockertools/manager.go

```diff
@@ -298,7 +298,7 @@ func (dm *DockerManager) inspectContainer(dockerID, containerName, tPath string)
 			if found {
 				data, err := ioutil.ReadFile(path)
 				if err != nil {
-					glog.Errorf("Error on reading termination-log %s: %v", path, err)
+					//glog.Errorf("Error on reading termination-log %s: %v", path, err)
```

Comment:
Ahh, someone breaks that feature which generates tons of logging messages. Fixed it. Thanks!